pkgs/by-name/ja/jailer/package.nix

```
@@ -0,0 +1,82 @@
+{
+  copyDesktopItems,
+  fetchFromGitHub,
+  lib,
+  makeDesktopItem,
+  makeWrapper,
+  stdenv,
+  wrapGAppsHook4,
+  ant,
+  jdk,
+  jre,
+  stripJavaArchivesHook,
+}:
+stdenv.mkDerivation rec {
+  name = "jailer";
+  version = "16.1.4";
+
+  src = fetchFromGitHub {
+    owner = "Wisser";
+    repo = "Jailer";
+    rev = "dcde48b2e0c7b68cc1b6f73c80fde06fdf31fdc6";
+    sha256 = "sha256-cPOWW2z44vCc/Fw1zYZUsYafgKr82e0g+K0Db5A9F5M=";
+  };
+
+
+  buildPhase = ''
+    runHook preBuild
+    rm jailer.jar
+    ant
+    runHook postBuild
+  '';
+
+  nativeBuildInputs = [ ant jdk stripJavaArchivesHook makeWrapper wrapGAppsHook4 copyDesktopItems ];
+
+  installPhase = ''
+    runHook preInstall
+
+    install -Dm644 jailer.jar $out/share/java/jailer.jar
+    install -Dm644 jailer-engine-16.1.4.jar $out/share/java/
```

Are you sure this is not versioned?
```suggestion
    install -Dm644 jailer-engine-${finalAttrs.version}.jar $out/share/java/
```